# Aquaculture_bacterial_indicators
Bacterial indicators for Mediterranean Aquaculture farms

A step towards the validation of bacteria biotic indices using DNA metabarcoding for benthic monitoring

Eva Aylagasa*, Javier Atalah<sup>b</sup>, Pablo SÃ¡nchez-Jerez<sup>c</sup>, John Pearman<sup>b</sup>, Nuria Casado<sup>c</sup>, Jorge Asensi<sup>c</sup>, Kilian Toledo-Guedes<sup>c</sup> and Susana Carvalho<sup>a</sup>
<sup>a</sup>King Abdullah University of Science and Technology (KAUST), Red Sea Research Center (RSRC), Biological and Environmental Sciences and Engineering (BESE), Thuwal 23955-6900, Saudi Arabia
<sup>b</sup>Cawthron Institute, Coastal and Freshwater Group, Nelson, 7010, New Zealand
<sup>c</sup>University of Alicante. Department of Marine Science and Applyed Biology. A.C. 99. 03080 Alicante. Spain.
*Corresponding author
Eva Aylagas: eva.aylagasmartinez@kaust.edu.sa


Keywords: biomonitoring, molecular methods, anthropogenic impacts, genomic approaches, metabarcoding

Abstract

Environmental genomic strategies are promising tools to monitoring biodiversity at a time of unprecedent impacts on marine environments. Efforts have increasingly been dedicated to the use of bacteria DNA derived data, relying on the fast generation times and high sensitivity to environmental disturbances, to develop biotic indices for benthic monitoring. However, 
a substantial debate exists whether bacteria-derived data using DNA metabarcoding should follow, for example, a taxonomy-based or a de novo approach in marine bioassessments. Here, we show case the value of DNA-based monitoring using the impact of fish farming as an example of environmental alterations in coastal areas. We analyzed samples collected immediately adjacent to the farm cages and along distance gradients from two aquaculture installations, as well as at control sites, to evaluate the effect of this activity on bacterial assemblages. Using the putative response of bacterial taxa to stress we calculated the taxonomy-based biotic index microgAMBI. Then, the distribution of individual Amplicon Sequence Variants (ASVs) as a function of a gradient in sediment acid volatile sulfides  was used to derive a de novo bacterial biotic index, specific for this dataset. Our results show that microgAMBI revealed a organically enriched environment along the gradient. However, the de novo biotic index outperformed microgAMBI by providing a higher discriminatory power in detecting changes in abiotic factors directly related to fish production, whilst allowing the identification of new ASVs bioindicators. The de novo strategy applied here represents a robust method to define new bioindicators in regions or habitats where no previous information about the response of bacteria to environmental stressors exists. 


